packages/headless/src/components/tooltip/tooltip.tsx

```diff
@@ -64,15 +71,30 @@ export const Tooltip = component$(
       })
     );
 
-    useClientEffect$(({ track }) => {
+    useClientEffect$(async ({ track }) => {
+      let cleanup = () => {
+        console.log('clean up signal has not been set');
```

Comment:
There are too many console.log in this file. While you probably need it for debugging, I don't see any value to commit them as part of the component implementation.

Reply:
Still a work in progress. All the console logs will be gone before we merge.